### ISSUE TITLE
Persistent Queue Delays with QiskitRuntimeService on 'yonsei-dedicated' Instance (ibm_yonsei Backend)

### ISSUE THREAD

**Issue by waizellovesjiyu** [Qiskit/documentation#2906]
Hi! This is my first time actually using IBM QPUs to process my code, and I need some help.

I'm experiencing persistent queue delays when submitting jobs to the `ibm_yonsei` backend via `QiskitRuntimeService` using the `yonsei-dedicated/internal/ymsphdcandid` instance. Jobs are successfully registered in the IBM Quantum Platform Workloads but remain in a `Queued` (Pending) state for extended periods (2+ hours for the latest job, 6+ hours for previous attempts), with no progress in execution. This issue has persisted across multiple job submissions despite code updates to align with the latest `SamplerV2` API in `qiskit-ibm-runtime==0.29.0`.

1. **Environment Setup**:
   - Qiskit IBM Runtime version: `0.29.0`
   - Python version: `3.11`
   - Platform: Google Colab
   - Instance: `yonsei-dedicated/internal/ymsphdcandid`
   - Backend: `ibm_yonsei`

2. **Code**:
   Below is the latest code used to submit the job:
   ```python
   import pennylane as qml
   import numpy as np
   from google.colab import drive
   from qiskit import QuantumCircuit, transpile
   from qiskit_ibm_runtime import QiskitRuntimeService, SamplerV2 as Sampler

   drive.mount('/content/drive')
   service = QiskitRuntimeService(
       channel='ibm_quantum',
       instance='yonsei-dedicated/internal/ymsphdcandid',
       token='-'
   )

   data = np.load('/content/drive/My Drive/wm811k/processed_data_10k.npz')
   X_train_embed = data['X_train']
   X_test_embed = data['X_test']
   y_train = data['y_train']
   y_test = data['y_test']
   X_train_cnn = X_train_embed[..., np.newaxis]
   X_test_cnn = X_test_embed[..., np.newaxis]

   from tensorflow.keras import layers, models
   cnn_model = models.Sequential([
       layers.Input(shape=(26, 26, 1)),
       layers.Conv2D(32, (3, 3), activation='relu', padding='same'),
       layers.MaxPooling2D((2, 2)),
       layers.Conv2D(64, (3, 3), activation='relu', padding='same'),
       layers.MaxPooling2D((2, 2)),
       layers.Flatten(),
       layers.Dense(16, activation='elu')
   ])
   X_train_embed = cnn_model.predict(X_train_cnn)
   X_test_embed = cnn_model.predict(X_test_cnn)

   dev = qml.device('default.qubit', wires=4, shots=1024)
   @qml.qnode(dev)
   def dv_qnn(inputs, weights):
       for i in range(4):
           qml.RY(inputs[i], wires=i)
       for layer in range(2):
           for i in range(4):
               qml.RZ(weights[layer, i, 0], wires=i)
               qml.RY(weights[layer, i, 1], wires=i)
           for i in range(3):
               qml.CNOT(wires=[i, i+1])
       return [qml.expval(qml.PauliZ(i)) for i in range(4)]

   def run_qnn_on_qpu(inputs, weights):
       with qml.tape.QuantumTape() as tape:
           dv_qnn(inputs, weights)
       qc = QuantumCircuit(4, 4)
       for op in tape.operations:
           if op.name == 'RY':
               qc.ry(float(op.parameters[0]), op.wires[0])
           elif op.name == 'RZ':
               qc.rz(float(op.parameters[0]), op.wires[0])
           elif op.name == 'CNOT':
               qc.cx(op.wires[0], op.wires[1])
       qc.measure(range(4), range(4))
       qc_transpiled = transpile(qc, backend=service.backend('ibm_yonsei'))
       sampler = Sampler(mode=service.backend('ibm_yonsei'))
       job = sampler.run([qc_transpiled], shots=1024)
       result = job.result()
       counts = result[0].data.c0.get_counts()
       total_shots = sum(counts.values())
       quasi_dist = {int(k, 2): v / total_shots for k, v in counts.items()}
       exp_vals = []
       for i in range(4):
           prob_0 = sum(quasi_dist.get(k, 0) for k in range(16) if not (k & (1 << i)))
           prob_1 = sum(quasi_dist.get(k, 0) for k in range(16) if k & (1 << i))
           exp_vals.append(prob_0 - prob_1)
       return np.array(exp_vals)

   weights_shape = (2, 4, 2)
   weights = qml.numpy.random.random(weights_shape, requires_grad=True)
   opt = qml.AdamOptimizer(stepsize=0.01)
   num_epochs = 10
   X_train_small = X_train_embed[:10, :4]
   y_train_small = y_train[:10, :4]

   for epoch in range(num_epochs):
       def cost_fn(w):
           preds = np.array([run_qnn_on_qpu(X_train_small[i], w) for i in range(len(X_train_small))])
           return qml.math.sum((preds - y_train_small)**2)
       weights, cost = opt.step_and_cost(cost_fn, weights)
       print(f"Epoch {epoch+1}/{num_epochs}, Cost: {cost}")

Did I miss anything? or Did I define anything wrong?
[job-cznpm6r112c0008b16n0-info.json](https://github.com/user-attachments/files/19544981/job-cznpm6r112c0008b16n0-info.json)

**Comment by kaldag** [comment]
@waizellovesjiyu you can contact Yonsei admin. If problem persists you could get in touch with IBM support through the admin.

**Comment by javabster** [comment]
Hi @waizellovesjiyu , the advice from @kaldag is correct, for queue time related things it is best to contact your admin and IBM support, this repository is for reporting issues related to documentation. Hope you manage to get your issue resolved :)

**Comment by Eric-Arellano** [comment]
Thanks @kaldag and @javabster! I'm going to close this as not planned since it's not docs related.
